packages/hawtio/src/plugins/camel/endpoints/BrowseMessages.tsx

```diff
@@ -40,13 +40,11 @@ import {
 import { eventService, NotificationType } from '@hawtiosrc/core'
 import { Position } from 'reactflow'
 import { MBeanNode } from '@hawtiosrc/plugins'
-
-const ForwardMessagesComponent: React.FunctionComponent<{
-  onForwardMessages: (uri: string, message?: MessageData) => void
-  currentMessage?: MessageData
-  endpoints: string[]
-}> = ({ onForwardMessages, currentMessage, endpoints }) => {
-  const [uri, setUri] = useState('')
+export const InputWithSuggestionsComponent: React.FunctionComponent<{
```

Comment:
If it's used from at least two places, why not extract it into a distinct file `InputWithSuggestions.tsx`?  And I think the name suffixes `...Component` is generally redundant. We don't use those suffixes in other places.